Description: Don't make beetsplug a namespace package
 This serves no purpose, if we are installing into a private location.
Author: Stefano Rivera <stefanor@debian.org>
Last-Updated: 2012-12-01
Forwarded: not-needed

--- a/setup.py
+++ b/setup.py
@@ -63,7 +63,6 @@
           'beetsplug.web',
           'beetsplug.lastgenre',
       ],
-      namespace_packages=['beetsplug'],
       entry_points={
           'console_scripts': [
               'beet = beets.ui:main',
